### ISSUE TITLE
Preserve user defined tab label, and not change it to vte window title

### ISSUE THREAD

**Issue by kikecalpe** [Guake/guake#42]
Preserve user defined tab label, and not change it to vte window title
http://guake.org/ticket/515
Reported by:    solarw  Owned by:   somebody
Priority:   trivial     Milestone:  0.5.0
Component:  guake   Version:  
Keywords:       Cc:  
Description

It's nice guake renames tabs according to vte window title, but i want so set my own persists labels to some titles. if i renaming tabs, they renamed by vte window title change. so i added patch to preserve my labels.

if i define label to any value except '%%title%%', label stays. if i not define any label, it will be changed according to vte window title. if i set label to '%%title%%' - it will be changed according to vte window title.

in summary: if you not set any label, it will be vte windows title. if you set label '%%title%%' it will be vte windows title. if you set any other label it will be preserved until manual change.
